main/zoe/api/zoe-contract-facet.md

```diff
@@ -313,9 +321,14 @@ Returns the `amountMath` object associated with the `brand` argument.
 ```js
 const assetAmountMath = zcf.getAmountMath(assetAmount.brand);
 ```
-## zcf.shutdown()
+## zcf.stopAcceptingOffers()
+- The contract requests Zoe to not accept offers for this contract instance. **tyg todo: No way of 
+starting it back up and accepting again?** It can't be called from outside the contract 
+unless the contract explicitly makes it accessible.
 
-**Note**: Still in development, use at your own risk.
+## zcf.shutdown(completion)
+
+**Note**: Still in development, use at your own risk. **tyg todo: Is this warning still valid?**
```

Comment:
This warning should be dropped. Chip's PR has finished the implementation, so this now does everything it's intended to.